### ISSUE TITLE
npath observations

### ISSUE THREAD

**Issue by achaubal** [hbutani/SQLWindowing#10]
hey Harish,

thanks for the npath function, 
a few observations,

i was running this:
from 
npath(<select  acct_no, my_dt ,my_score  
               from  my_tab>
partition by my_acct_no 
order     by my_dt, 
'a.b.c',
<["a" : "my_score == 444","b" : "my_score == 555","c" : "my_score == 666"]>,
<[ "distinct my_acct_no" ]>)
select my_acct_no into path='npath_10_2011_09';

found that it doesn't like distinct in the final select. In this particular case, may be it doesn't matter since the partition is by the acct number. But how can I do Aster style distinct in the "outer" select?

**Comment by hbutani** [comment]
yes there is no distinct possible in the result column expressions
What we can do is add another table function that does the post processing for you. So you would have to write:

from someFn(npath(...))
partition by...

Tell me more about your use case; then we can try to come up with a solution.
